bin/java

```diff
@@ -111,6 +111,31 @@ Heroku"
   fi
 }
 
+download_binary() {
+  local jdkUrl="${1:?}"
+  local jdkTarball="${2:?}"
+  local curDir=${JVM_COMMON_DIR:-$(cd $(dirname ${BASH_SOURCE[0]}) && cd .. && pwd )}
+
+  curl --retry 3 --silent --show-error --location "${jdkUrl}" --output "${jdkTarball}"
```

Comment:
https://github.com/heroku/heroku-buildpack-php/blob/dbfaecee08f1286c095c69432e71d1a0b6c7df68/bin/util/common.sh#L121-L129

Reply:
I'm working on adding that function in #103

Reply:
I think that should be merged first, otherwise there will be random failures (this does happen often enough that it will occasionally cause failures).